scripts/detection/ssd/train_ssd.py

```diff
@@ -105,6 +129,72 @@ def get_dataloader(net, train_dataset, val_dataset, data_shape, batch_size, num_
         batch_size, False, batchify_fn=val_batchify_fn, last_batch='keep', num_workers=num_workers)
     return train_loader, val_loader
 
+def to_normalized_ltrb_list(anchors, size):
+    anchors_np = anchors.squeeze().asnumpy()
+    anchors_np_ltrb = anchors_np.copy()
+    anchors_np_ltrb[:, 0] = anchors_np[:, 0] - 0.5 * anchors_np[:, 2]
+    anchors_np_ltrb[:, 1] = anchors_np[:, 1] - 0.5 * anchors_np[:, 3]
+    anchors_np_ltrb[:, 2] = anchors_np[:, 0] + 0.5 * anchors_np[:, 2]
+    anchors_np_ltrb[:, 3] = anchors_np[:, 1] + 0.5 * anchors_np[:, 3]
+    anchors_np_ltrb /= size
+    return anchors_np_ltrb.flatten().tolist()
+
+def get_dali_dataloader(net, dataset_name, data_shape, global_batch_size, num_workers, devices, file_root, ctx, horovod):
```

Comment:
here in the script, there are too many hardcoded paths, similar to gluoncv's original dataset, I think DALI dataset reader should also follow the same tradition, e.g., in https://github.com/dmlc/gluon-cv/tree/master/gluoncv/data/mscoco/detection.py , we can have a `DALICOCODetection` class which handles the dataset location and `coco_size`. It will be shared by all detection algorithms.

Reply:
@zhreshold this and previews comments addressed in https://github.com/dmlc/gluon-cv/pull/807/commits/7be871ea66de988055cf1359d1c831a0bd532f7c